adding rapid7 to run tasks partner list (#160)
hashicorp/terraform-docs-common: website/docs/cloud-docs/integrations/run-tasks/index.mdx

To get started, follow the instructions in the [Terraform Cloud integration](https://docs.oak9.io/oak9/fundamentals/integrations/setup-terraform-cloud-run-task) user documentation.

### Rapid7

> [Rapid7's](https://rapid7.com/) InsightCloudSec enables security teams to provide a single, consistent definition of good across the full CI/CD pipeline, allowing them to evaluate potential risks of Infrastructure as Code (IaC) templates as they are built and pre-deployment.

Reviewer: ```suggestion
[Rapid7's](https://rapid7.com/) InsightCloudSec enables security teams to provide a single, consistent definition of good across the full CI/CD pipeline, allowing them to evaluate potential risks of Infrastructure as Code (IaC) templates during building and pre-deployment.
```

Suggest rewording "as they are built and pre-deployment" to "during building and pre-deployment" to avoid passive voice and maintain phrasing consistency.
